### PR TITLE
Update index_en.rst

### PR DESCRIPTION
**Pull request by legend06hvl**
Update en version
fix https://github.com/PaddlePaddle/Paddle/issues/8915

### CODE REVIEW

File: doc/v2/howto/capi/index_en.rst [hunk 1]
```diff
@@ -1,6 +1,23 @@
 C-API Prediction Library
 ========================
 
+After we trained a neural network model, then we use it to make predictions. Prediction is a process of preparing input data and processing the model to get the result.
```
Comment (abhinavarora) [inline doc/v2/howto/capi/index_en.rst]: trained -> **train**

Comment (abhinavarora) [reply doc/v2/howto/capi/index_en.rst]: I think we should replace the word prediction with Inference. The title can also read C-API Inference Library.

Comment (abhinavarora) [reply doc/v2/howto/capi/index_en.rst]: The sentence should be as follows:

After we train a neural network, we use it to do inference. Inference is the process of preparing input data and propagating it through the model to produce the result.

File: doc/v2/howto/capi/index_en.rst [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+
+Compared with model training, prediction has the following features:
+
+#. Prediction does not require backpropagation and parameter updates during training.
```
Comment (abhinavarora) [inline doc/v2/howto/capi/index_en.rst]: Inference does not require backpropagation and parameter updates, as required during training.

File: doc/v2/howto/capi/index_en.rst [hunk 3]
```diff
@@ -1,6 +1,23 @@
...
+#. Labels are not needed in prediction.
+#. Most of the time, predictions need to be integrated with the user system.
+
+Therefore, the model prediction SDK needs to be designed separately and has the following features:
+
+#. The predictive SDK does not include reverse propagation and parameter updates to reduce the size of the SDK.
```
Comment (abhinavarora) [inline doc/v2/howto/capi/index_en.rst]: reverse propagation -> **backpropagation**

File: doc/v2/howto/capi/index_en.rst [hunk 4]
```diff
@@ -1,6 +1,23 @@
...
+#. The predictive SDK needs a simple user interface for ease of use.
+#. Since the input data may have a variety of structures, the format of the input data is clearly and compactly packaged.
+#. In order to be compatible with the user's system, the SDK's interface must conform to the C-standard interface.
```
Comment (abhinavarora) [inline doc/v2/howto/capi/index_en.rst]: with the user's system -> **with user's system**

File: doc/v2/howto/capi/index_en.rst [hunk 5]
```diff
@@ -1,6 +1,23 @@
...
+
+PaddlePaddle provides C-API to solve the above problem. The use of C-API, we provide the following guidelines:
```
Comment (abhinavarora) [inline doc/v2/howto/capi/index_en.rst]: Following are the guidelines to use the C-API:

**Comment by shanyi15** [comment]
Thank you so much! @abhinavarora

**Pull request merged**